cypress/e2e/focus-trap-demo.cy.js

```diff
@@ -1655,6 +1655,39 @@ describe('focus-trap', () => {
     });
   });
 
+  describe('demo: multiple traps with manual pause', () => {
+    it('manually paused trap will not be automatically unpaused if another trap is deactivated', () => {
+      cy.get('#demo-multipletraps-manual-pause').as('testRoot');
+
+      // activate trap 1
+      cy.get('@testRoot')
+        .findByRole('button', { name: /^activate trap 1/ })
+        .as('lastlyFocusedElementBeforeTrapIsActivated')
+        .click();
+
+      // activate focus trap 2. This should pause trap 1
+      cy.get('@testRoot')
+        .findByRole('button', { name: /^activate trap 2/ })
+        .click();
+
+      // pause focus trap 1 manually
+      cy.get('@testRoot')
+        .findByRole('button', { name: /^pause trap 1/ })
+        .click();
+
+      // stop focus trap 2
+      cy.get('@testRoot')
+        .findByRole('button', { name: /^deactivate trap 2/ })
+        .click({ force: true }); // click fails without `force` option
+
+      // focus can be transitioned freely when trap 2 is deactivated because trap 1 is still paused
+      cy.findByRole('heading', { name: 'focus-trap demo' })
+        .as('outsideEl')
+        .click();
+      verifyFocusIsNotTrapped(cy.get('@outsideEl'));
+    });
+  });
```

Comment:
Added a test, but it always succeeds even without the fix. I noticed that when running the E2E test locally, the focus trap never works.

Reply:
Hmmm.. That's a bit strange. I'll try to check it out locally myself.

Reply:
If I remove the new check on line 36:
```
// !trapStack[trapStack.length - 1]._isManuallyPaused()
```
and try to run the e2e tests locally (`npm run test:e2e:chrome`), then the test fails with this error:
```
AssertionError: Timed out retrying after 4000ms: expected '<h1>' to be 'focused'
```
which I take to indicate that the final assertion, "focus can be transitioned freely when trap 2 is deactivated because trap 1 is still paused", fails because trap 1 got auto-unpaused since we're no longer checking for the manually-paused state.

Reply:
Oh, I ran the test with `npm run test:e2e:dev`. Using `npm run test:e2e:chrome` works.